lib/iris/tests/test_basic_maths.py

```diff
@@ -494,15 +494,6 @@ def test_multiplication_not_in_place(self):
         b = iris.analysis.maths.multiply(a, 5, in_place=False)
         self.assertIsNot(a, b)
 
-    def test_type_error(self):
```

Comment:
I think the `TypeError` raised comes from NumPy in the end. But having taken a quick look I agree with your outlook that they're not necessary - if it were our own error case we would want to test it, but it is no surprise that if you try and mulitply a cube by a string it won't work!